Tribler/community/gigachannel/community.py

```diff
@@ -89,13 +106,84 @@ def on_blob(self, peer, blob):
         if reply_blob:
             self.endpoint.send(peer.address, self.ezr_pack(self.NEWS_PUSH_MESSAGE, RawBlobPayload(reply_blob)))
 
+    def send_search_request(self, query_filter, metadata_type='', sort_by="HEALTH", sort_asc=0, hide_xxx=True):
+        """
+        Sends request to max_search_peers from peer list. The request is cached in request cached. The past cache is
+        cleared before adding a new search request to prevent incorrect results being pushed to the GUI.
+        """
+        search_candidates = self.get_peers()[:max_search_peers]
+        search_request_cache = SearchRequestCache(self.request_cache, search_candidates)
+        self.request_cache.clear()
+        self.request_cache.add(search_request_cache)
+
+        search_request_payload = SearchRequestPayload(search_request_cache.number, query_filter.encode('utf8'),
+                                                      metadata_type, sort_by, sort_asc, hide_xxx)
+        self._logger.debug("Started remote search for query:%s", query_filter)
```

Comment:
I think this can be on INFO level. Minor comment: please add a space after the `:`.